jso/apis/src/main/java/org/teavm/jso/webaudio/AudioContext.java

```diff
@@ -0,0 +1,164 @@
+/*
+ *  Copyright 2015 Alexey Andreev.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.jso.webaudio;
+
+import org.teavm.jso.JSBody;
+import org.teavm.jso.JSMethod;
+import org.teavm.jso.JSObject;
+import org.teavm.jso.JSProperty;
+import org.teavm.jso.dom.events.EventListener;
+import org.teavm.jso.dom.html.HTMLMediaElement;
+import org.teavm.jso.typedarrays.ArrayBuffer;
+import org.teavm.jso.typedarrays.Float32Array;
+
+/**
+ *
+ */
+public abstract class AudioContext implements JSObject {
+    String STATE_SUSPENDED = "suspended";
+    String STATE_RUNNING = "running";
+    String STATE_CLOSE = "close";
+
+    @JSProperty
+    public abstract AudioDestinationNode getDestination();
+
+    @JSProperty
+    public abstract float getSampleRate();
+
+    @JSProperty
+    public abstract double getCurrentTime();
+
+    @JSProperty
+    public abstract AudioListener getListener();
+
+    @JSProperty
+    public abstract String getState();
+
+    @JSProperty("onstatechange")
+    public abstract void setOnStateChange(EventListener listener);
```

Comment:
I see. Fixed this errors and fixed travis configuration. Please, merge with the latest master and build your branch with maven. You must check whether build passes without error.